lib/phoenix_live_view/helpers.ex

```diff
@@ -1113,7 +1113,7 @@ defmodule Phoenix.LiveView.Helpers do
     # unless the action is given.
     {attrs, hidden_method, csrf_token} =
       if action do
-        {method, opts} = Keyword.pop(opts, :method)
+        {method, opts} = Keyword.pop(opts, :method, "post")
```

Comment:
One way to reproduce this is creating a test with `action="/"` without a http method in here:

https://github.com/phoenixframework/phoenix_live_view/blob/master/test/phoenix_live_view/helpers_test.exs#L90

Something like this:

```eex
      html =
        parse(~H"""
        <.form :let={f} for={:myform} action="/">
          <%= text_input f, :foo %>
        </.form>
        """)
```

Would raise the error you are facing so we can cover this up :)

Reply:
> One way to reproduce this is creating a test with `action="/"` without a http method in here:
> 
> https://github.com/phoenixframework/phoenix_live_view/blob/master/test/phoenix_live_view/helpers_test.exs#L90
> 
> Something like this:
> 
> ```
>       html =
>         parse(~H"""
>         <.form :let={f} for={:myform} action="/">
>           <%= text_input f, :foo %>
>         </.form>
>         """)
> ```
> 
> Would raise the error you are facing so we can cover this up :)

@feliperenan Indeed, that's the test that triggers the error :) Thanks!